client/my-sites/sidebar-unified/style.scss

```diff
@@ -558,6 +558,15 @@ $font-size: rem( 14px );
 // Flyout menu (showing from >782px)
 @media screen and ( min-width: 783px ) {
 	.is-nav-unification {
+		.focus-content,
+		.focus-sites,
+		.focus-sidebar {
+			.sidebar,
```

Comment:
Should we add a new entry to client/assets/stylesheets/shared/functions/_z-index.scss (with the same z-index) to account for the sidebar, since this one is called `layout__secondary`?

Reply:
Sorry, just now seeing that this was a [pre-existing issue in the code](https://github.com/Automattic/wp-calypso/blob/acb5ce33d083470186883dff26cb280692147e67/client/my-sites/sidebar-unified/style.scss#L571).

Feel free to ignore the comment above unless you really want to fix it.

Reply:
I understand that it would be cleaner, but since all this layers normally are in the same z layer I don't see what would be the benefit. I think we should touch that `z-index.scss` file when/if the z-index stacking changes.